projects/storefrontlib/tsconfig.lib.json

```diff
@@ -14,7 +14,29 @@
     "types": [],
     "lib": ["es2020", "dom"],
     "paths": {
-      "@spartacus/core": ["dist/core"]
+      "@spartacus/core": ["dist/core"],
+      "@spartacus/cart/main/components": ["dist/cart/main/components"],
```

Comment:
This file is auto-generated. We will remove `add-to-cart` and `CartSharedModule` later.